### ISSUE TITLE
using in polygon

### ISSUE THREAD

**Issue by weesleei** [gemworks/gem-farm#39]
I would like to know, if I edited, would it be possible to change the network to polygon, I have an nft project and I needed a staking system in polygon

**Comment by kafinsalim** [comment]
No, It's on different L1 network and polygon smart contract written in different language
